app/build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     compile('com.mapzen.android:lost:3.0.2')          // used for Location Acquisition (GPS)
 
     // note: the MapQuest Mapping SDK (i.e. "MapView") used to display our route to navigate
-    compile('com.mapquest:mapping-android-sdk:1.6.3') {
+    compile('com.mapquest:mapping-android-sdk:2.0.1') {
```

Comment:
cool thx -- yeah and maybe let's use the (revised and shortened) comments per the new getting-started doc? Here they are again:
```
    compile('com.mapquest:navigation:3.2.2')            // the MapQuest Navigation SDK
    compile('com.mapzen.android:lost:3.0.4')            // used for GPS Location Acquisition (see notes)

    compile('com.mapquest:mapping-android-sdk:2.0.1') { // for Mapping SDK MapView; displays route to navigate
        exclude group: 'com.android.support'            // (note: in your app, you might omit this 'exclude')
    }

    compile('com.mapquest:search-ahead-v3:1.2.9')       // for the MapQuest "Search Ahead" feature (optional)
```